<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QWidget" name="VBox_Widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="Item_Clear_BTN">
          <property name="text">
           <string>清空镜头</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Item_Add_BTN">
          <property name="text">
           <string>添加镜头</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="Cam_Base_Layout">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="Cam_Base_Label">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>基准镜头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Cam_LE">
           <property name="text">
            <string>Cam_Base</string>
           </property>
           <property name="placeholderText">
            <string>镜头名称</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="Cam_Item_Scroll">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="Item_Layout">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>362</width>
           <height>110</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="Button_Layout" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="Select_Path_BTN">
             <property name="text">
              <string>选择所有MotionPath</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Batch_Keyframe_BTN">
             <property name="text">
              <string>批量偏移MotionPath关键帧</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="Batch_Position_BTN">
           <property name="text">
            <string>批量匹配镜头组位置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Batch_Constraint_BTN">
           <property name="text">
            <string>批量添加约束</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Con_Keyframe_BTN">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:red</string>
        </property>
        <property name="text">
         <string>设置约束关键帧</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
